bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -7890,12 +7901,25 @@ private void selectTimeSlice(final SelectionChartInfo chartInfo) {
       final Table table = (Table) _timeSlice_Viewer.getControl();
       final int itemCount = table.getItemCount();
 
-      //TODO FB remove this property ??
-      _lastSelectedChartInfo = chartInfo;
-      if (chartInfo.isSelectInBetweenTimeSlices) {
-         final int minSelectedValue = Math.min(chartInfo.leftSliderValuesIndex, chartInfo.rightSliderValuesIndex);
-         final int maxSelectedValue = Math.max(chartInfo.leftSliderValuesIndex, chartInfo.rightSliderValuesIndex);
-         table.setSelection(minSelectedValue, maxSelectedValue);
+      if (_prefStore.getBoolean(
+            ITourbookPreferences.TOGGLE_STATE_SELECT_INBETWEEN_TIME_SLICES)) {
+
+         // delay the selection of multiple lines, moving the mouse can occur very often
+         _parent.getDisplay().timerExec(50, new Runnable() {
+
+            @Override
+            public void run() {
+
+               if (_parent.isDisposed()) {
+                  return;
+               }
+
+               final int minSelectedValue = Math.min(chartInfo.leftSliderValuesIndex, chartInfo.rightSliderValuesIndex);
+               final int maxSelectedValue = Math.max(chartInfo.leftSliderValuesIndex, chartInfo.rightSliderValuesIndex);
+               table.setSelection(minSelectedValue, maxSelectedValue);
+            }
+         });
```

Comment:
The only reason why a delayed execution is performed, is not applied from https://github.com/wolfgang-ch/mytourbook/pull/192#discussion_r443194322 
-> The delayed execution should skip all selection events which occurred in the last 50 ms, therefor the `__runningId` is set and checked